app/site.rb

```diff
@@ -117,6 +117,12 @@ def git?
 			cache_path.join('.git').directory?
 		end
 
+		def log
+			git.log.tap(&:size)
```

Comment:
What is `git.log` when there are no entries? If it's nil we could return `Array(git.log)`.